neo/Network/P2P/RemoteNode.cs

```diff
@@ -25,14 +25,13 @@ internal class Relay { public IInventory Inventory; }
         private BloomFilter bloom_filter;
         private bool ack = true;
         private bool verack = false;
+        private int listenerTcpPort = 0;
 
         public IPEndPoint Listener => new IPEndPoint(Remote.Address, ListenerTcpPort);
-        public override int ListenerTcpPort => Version?.Capabilities
-            .Where(u => u.Type == NodeCapabilities.TcpServer)
-            .Cast<ServerCapability>()
-            .First()?.Port ?? 0;
+        public override int ListenerTcpPort => listenerTcpPort;
         public VersionPayload Version { get; private set; }
-        public uint LastBlockIndex { get; private set; }
+        public bool AcceptRelay { get; private set; } = false;
```

Comment:
Is it necessary to keep AcceptRelay?

Reply:
We can deduce that if is not full node, accept relay

Reply:
Can we just check the `FullNodeCapability`?

Reply:
But light nodes could be able to relay

Reply:
A node should verify the transaction before relaying. A light node cannot verify a transaction.

Reply:
Then i will remove `AcceptRelay`